Essentials/src/config.yml

```diff
@@ -844,6 +844,9 @@ protect:
     # Should Wither damage be disabled?
     wither: false
 
+    #Should sign dyeing be disabled?
```

Comment:
Put space before "#" signal, to the text is be readable.

Reply:
Solved